chrome/content/zotero/zoteroPane.xul

```diff
@@ -370,186 +370,187 @@
 								onkeydown="ZoteroPane_Local.handleKeyDown(event, this.id)"
 								onselect="ZoteroPane_Local.itemSelected(event)"
 								oncommand="ZoteroPane_Local.serializePersist()"
-								flex="1">
+								flex="1"
+								zotero-persists="current-view-group">
 								<treecols id="zotero-items-columns-header">
 									<treecol
-										id="zotero-items-column-title" primary="true"
+										id="zotero-items-column-title" primary="true" default-in="default feed"
 										label="&zotero.items.title_column;"
 										flex="4" zotero-persist="width ordinal hidden sortActive sortDirection"/>
 									<splitter class="tree-splitter"/>
 									<treecol
-										id="zotero-items-column-firstCreator"
+										id="zotero-items-column-firstCreator" default-in="default"
```

Comment:
You've been looking at more feeds, but I'd guess that enough have creators that it would make sense to show this by default in feeds.